ObjcVoiceQuickstart/ViewController.m

```diff
@@ -686,4 +713,46 @@ - (void)performAnswerVoiceCallWithUUID:(NSUUID *)uuid
     }
 }
 
+#pragma mark - Ringtone
+
+- (void)playRingback {
+    NSString *ringtonePath = [[NSBundle mainBundle] pathForResource:@"ringtone" ofType:@"wav"];
+    if ([ringtonePath length] <= 0) {
+        NSLog(@"Can't find sound file");
+        return;
+    }
+    
+    NSError *error;
+    self.ringtonePlayer = [[AVAudioPlayer alloc] initWithContentsOfURL:[NSURL URLWithString:ringtonePath] error:&error];
+    if (error != nil) {
+        NSLog(@"Failed to initialize audio player: %@", error);
+    } else {
+        self.ringtonePlayer.delegate = self;
+        self.ringtonePlayer.numberOfLoops = -1;
+        
+        self.ringtonePlayer.volume = 1.0f;
+        [self.ringtonePlayer play];
+    }
+}
+
+- (void)stopRingback {
+    if (!self.ringtonePlayer.isPlaying) {
+        return;
+    }
+    
+    [self.ringtonePlayer stop];
+}
+
+- (void)audioPlayerDidFinishPlaying:(AVAudioPlayer *)player successfully:(BOOL)flag {
+    if (flag) {
+        NSLog(@"Audio player finished playing successfully");
+    } else {
+        NSLog(@"Something wrong with audio player playback");
```

Comment:
Change the log messages to - 
`Audio player finished playing the ringback tone successfully`
and
`Audio player finished playing the ringback tone unsuccessfully`